README.md

```diff
@@ -1,12 +1,81 @@
 
 This is the API used for Email orchestration between CLA services and GOV.UK Notify.
 
+## Development in a virtual environment
+
```

Comment:
I am wondering if we should list the python version and other major things that need their version listed at the top in one place so when this changes we dont miss anything.